tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -19,8 +19,11 @@ def __init__(self, marionette):
         self.apps = GaiaApps(self.marionette)
         self.accessibility = Accessibility(self.marionette)
         self.frame = None
-        self.manifest_url = hasattr(self, 'manifest_url') and self.manifest_url or None
```

Comment:
I don't think @mwargers because we use it on the launch function : 
def launch(self, launch_timeout=None):
        self.app = self.apps.launch(self.name, self.manifest_url, self.entry_point, launch_timeout=launch_timeout)